src/components/editMeasure/measureDetails/MeasureDetails.test.tsx

```diff
@@ -43,6 +46,10 @@ MeasureDisclaimerMock.mockImplementation(() => {
   return <div>Mock Measure Disclaimer</div>;
 });
 
+MeasureRationaleMock.mockImplementation(() => {
+  return <div>Mock Measure Rationale</div>;
+});
+
```

Comment:
jest.mock"./measureRationale/MeasureRationale", () => () => (
  <div>Mock Measure Rationale</div>
));

Wondering if we can mock and return an element immediately. The only advantage I am seeing is that we could reduce number of lines of code.

Reply:
Thanks Rohit. I tried it, didn't work, I get "Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object." error.